_includes/download/LeapMicro.html

```diff
@@ -0,0 +1,14 @@
+{% include choosing_media.html %}
+{% include documentation.html %}
+<section class="py-3">
+  <div class="container">
+    <h3 id="recommended-system-requirements">{{ locale.requirements }}</h3>
```

Comment:
Please have a look if you don't want to copy MicroOS here instead, since I assume it will have similar system requirements, not like Leap.